_posts/2020-08-06-virtual-meeting.md

```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+title: "Virtual Meeting"
+date: 2020-08-06 09:30
+categories: meetings
+---
+
+## Meta:
+
+- Location: [Zoom](https://z.umn.edu/cpmstream)
+- Day: Thursday, August 6
+- Time: 9:30
+
+## Agenda:
+
+- Introductions
+- How We Built It: ChimeIn 2, Colin McFadden
+- Lightning Talks
+  - New "SQL Advisor" Tool - Brad Carlson, OIT
+  - Oracle Database Upgrade Program - Rafael Santos, OIT
+
+## Talk Descriptions:
```

Comment:
Include the description for the feature talk here:

### How We Built It: ChimeIn 2
Colin McFadden, LATIS

>ChimeIn 2 is an all new version of CLA's popular "ChimeIn" student response application. It allows instructors get realtime feedback from students, similar to other "clicker" systems like iClicker or TopHat. ChimeIn is totally free and fun to use. ChimeIn 2 is built with the Laravel PHP framework, a VueJS frontend, and heavy use of websockets for realtime interaction. Although it's currently deployed with Ansible on JME hosts, we've also got a nice container environment suitable for both dev and prod.